src/runtime/components/AntAccordionItem.vue

```diff
@@ -35,9 +38,18 @@ function onClick() {
       <div
         class="hover:text-gray-800 flex justify-between items-center"
       >
-        <span class="text-sm font-semibold">
-          {{ label }}
-        </span>
+        <div class="flex items-center gap-2">
+          <AntIcon
+            v-if="iconLeft"
+            :size="IconSize.sm"
+            :icon="faQuestionCircle"
```

Comment:
@beyer-johannes Should this Icon be changable from the outside? 
For example: Is it possible that we need a warn triangle instead of this question circle here? 

Also: maybe it would make sense to provide a Slot here to overwrite the Icon with something else

Reply:
Yes make sense, but not needed yet. I whould keep it simple.